### DISCUSSION TITLE
Is there support for net new SYSTEM property feature of fetch contents?

### DISCUSSION THREAD

**Discussion by ClausKlein** [cpm-cmake/CPM.cmake#438] [Ideas]
New in version `CMake v3.25`:

**SYSTEM**
If the **SYSTEM** argument is provided, the [SYSTEM](https://cmake.org/cmake/help/latest/prop_dir/SYSTEM.html#prop_dir:SYSTEM) directory property of a subdirectory added by [FetchContent_MakeAvailable()](https://cmake.org/cmake/help/latest/module/FetchContent.html#command:fetchcontent_makeavailable) will be set to true. This will affect non-imported targets created as part of that command. See the [SYSTEM](https://cmake.org/cmake/help/latest/prop_tgt/SYSTEM.html#prop_tgt:SYSTEM) target property documentation for a more detailed discussion of the effects.

**Comment by TheLartians** [comment]
Hey thanks for the heads up! I think it would be a great idea to include this feature in `cpm_add_subdirectory` if possible.

**Reply by ClausKlein** [reply]
done with https://github.com/cpm-cmake/CPM.cmake/releases/tag/v0.38.0
